read-the-docs/source/rt.rst

```diff
@@ -16,13 +16,32 @@ https://rt-wiki.bestpractical.com/wiki/REST
 Methods
 ^^^^^^^
 
+-  `get_ticket_content() <#get_ticket_content>`__
 -  `create() <#create>`__
 -  `edit() <#edit>`__
 -  `add_comment() <#comment>`__
 -  `change_status() <#status>`__
 -  `add_attachment() <#add_attachment>`__
 
 
+get_ticket_content()
+--------------------
+
+``get_ticket_content(self, ticket_id=None, option='show')``
+
+Queries the RT API to get the ticket_content using ticket_id. The
```

Comment:
Update this section to explain that show, attachments, and history return data in a dictionary format, while comment returns a list of strings.

Reply:
Updated.